src/pages/index.js

```diff
@@ -141,32 +141,32 @@ function Home() {
             <div className={styles.heroInner}>
               <h1 className={styles.heroProjectTagline}>
                 <img
-                  alt="Demisto"
+                  alt="Cortex XSOAR"
                   className={styles.heroLogo}
-                  src={useBaseUrl("img/Demisto-Icon-White.png")}
+                  src={useBaseUrl("img/Cortex-XSOAR-black.svg")}
                 />
                 Automate the{" "}
                 <span className={styles.heroProjectKeywords}>future</span> with{" "}
-                <span className={styles.heroProjectKeywords}>Demisto.</span>{" "}
+                <span className={styles.heroProjectKeywords}>Cortex XSOAR</span>{" "}
               </h1>
-              <h3>
+              <h3 className={styles.heroProjectDescription}>
                 Develop new integrations, automations, playbooks, reports and
                 more...
               </h3>
               <div className={styles.indexCtas}>
                 <Link
                   className={classnames(
-                    "button button--outline button--secondary button--lg",
-                    styles.indexCtasGetStartedButton
+                    "button button--info button--secondary button--lg",
+                    styles.headerButtons
                   )}
-                  to="docs/why-demisto"
+                  to="docs/partners/why-xsoar"
```

Comment:
To be consistent with the rest please point it to `/docs/partners/why-xsoar` with the heading `/`